[BUG] agents-copilotstudio-client breaking changes in browser bundle and TypeScript types

Smoke testing `@microsoft/agents-copilotstudio-client@next` (`1.5.0-beta.26.gaab35a7b76`) against the samples in this repo revealed a breaking change that require users to modify working code.

## Top-level `await` in `browser.mjs` breaks esbuild bundling

  ### Affected sample
  - `samples/nodejs/copilotstudio-webchat-react`

  ### Error
```
> copilotstudio-webchat-react@0.1.0 build
> esbuild src/index.tsx --bundle --platform=browser --target=esnext --outdir=public/dist --jsx=automatic

✘ [ERROR] Top-level await is currently not supported with the "iife" output format

    node_modules/@microsoft/agents-copilotstudio-client/dist/src/browser.mjs:5:13983:
      5 │ ...e:e?Rs(e):vo,metric:e?Ps(e):As()}}var{SpanNames:Ge,MetricNames:re,debug:Et,trace:H,metric:ne}=await js({otel:()=>Promise.resolve().then(()=>(Fa(),za)),logs:()=>Promise.resolve().then(()=>(Xa()...
        ╵                                                                                                  ~~~~~

✘ [ERROR] This require call is not allowed because the transitive dependency "node_modules/@microsoft/agents-copilotstudio-client/dist/src/browser.mjs" contains a top-level await

    src/Chat.tsx:19:29:
      19 │     agentsSettings = require('./settings.js').settings
         ╵                              ~~~~~~~~~~~~~~~

  The file "src/settings.js" imports the file
  "node_modules/@microsoft/agents-copilotstudio-client/dist/src/browser.mjs" here:

    src/settings.js:8:35:
      8 │ import { ConnectionSettings } from '@microsoft/agents-copilotstudio-client'
        ╵                                    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

  The top-level await in "node_modules/@microsoft/agents-copilotstudio-client/dist/src/browser.mjs"
  is here:

    node_modules/@microsoft/agents-copilotstudio-client/dist/src/browser.mjs:5:13983:
      5 │ ...e:e?Rs(e):vo,metric:e?Ps(e):As()}}var{SpanNames:Ge,MetricNames:re,debug:Et,trace:H,metric:ne}=await js({otel:()=>Promise.resolve().then(()=>(Fa(),za)),logs:()=>Promise.resolve().then(()=>(Xa()...
        ╵                                                                                                  ~~~~~
```


  ### Root cause
  `browser.mjs` now contains a top-level `await` (appears to be from OpenTelemetry initialization). esbuild defaults to `iife` output format, which does not support top-level await. The build fails entirely
  with no workaround in the existing sample config.

  ### Fix requested
  Avoid top-level `await` in the browser bundle

--- a/packages/agents-copilotstudio-client/package.json
+++ b/packages/agents-copilotstudio-client/package.json
@@ -24,7 +24,7 @@
     "crypto": "./src/browser/crypto.ts"
   },
   "scripts": {
-    "build:browser": "esbuild --platform=browser --target=es2022 --format=esm --bundle --sourcemap --minify --outfile=dist/src/browser.mjs src/index.ts"
+    "build:browser": "esbuild --platform=browser --target=es2019 --format=esm --bundle --sourcemap --minify --outfile=dist/src/browser.mjs src/index.ts"
   },
   "dependencies": {
     "@microsoft/agents-activity": "file:../agents-activity",

--- a/packages/agents-telemetry/package.json
+++ b/packages/agents-telemetry/package.json
@@ -22,7 +22,7 @@
   "main": "./dist/cjs/src/index.cjs",
   "module": "./dist/esm/src/index.mjs",
   "types": "./dist/esm/src/index.d.mts",
-  "browser": "./dist/esm/src/index.mjs",
+  "browser": "./dist/esm/src/index.cjs",
   "scripts": {
     "clean:esm": "node -e \"require('node:fs').rmSync('dist/esm', { recursive: true, force: true })\"",
     "build:esm": "tsc --project tsconfig.esm.json && node scripts/esm.mjs"
@@ -56,7 +56,7 @@
     ".": {
       "import": {
         "types": "./dist/esm/src/index.d.mts",
-        "browser": "./dist/esm/src/index.mjs",
+        "browser": "./dist/esm/src/index.cjs",
         "default": "./dist/esm/src/index.mjs"
       },
       "require": {

--- a/packages/agents-telemetry/tsconfig.esm.json
+++ b/packages/agents-telemetry/tsconfig.esm.json
@@ -1,5 +1,6 @@
 {
   "extends": "./tsconfig.json",
+  "exclude": [],
   "compilerOptions": {
     "outDir": "dist/esm",
     "composite": false,

--- a/packages/agents-telemetry/tsconfig.esm.json
+++ b/packages/agents-telemetry/tsconfig.esm.json
@@ -1,8 +1,5 @@
 {
   "extends": "./tsconfig.json",
-  "exclude": [
-    "src/**/*.cts"
-  ],
   "compilerOptions": {
     "outDir": "dist/esm",
     "composite": false,

